sonar-project.properties

```diff
@@ -0,0 +1 @@
+sonar.exclusions= **/*.spec.ts, **/*_test.go
```

Comment:
Would sonar tolerate a newline at the end of the file? 😉 

(Also, is it currently skipping `.github/`?)

Reply:
sure, adding a newline won't do any harm ;) SCM directories such as `.github` are apperently excluded per default (see the description of the `sonar.scm.exclusions.disabled` property at https://sonarcloud.io/documentation/analysis/analysis-parameters/)